src/components/org/PositionCard/components/PositionInstance.tsx

```diff
@@ -8,6 +8,8 @@ import PositionTimeline from './PositionTimeline';
 type PositionInstanceProps = {
     position: Position;
     instance?: PositionInstance;
+    isFuture?: boolean;
```

Comment:
Do we need these props in the Instance component?

Reply:
Removed